src/transformState.js

```diff
@@ -4,8 +4,29 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  const trans = state;
+
+  for (const a in actions) {
```

Comment:
```suggestion
  for (const action in actions) {
```
Let's provide full, descriptive names